oauth2-server/src/main/java/com/clouway/oauth2/ClientAuthorizationActivity.java

```diff
@@ -49,7 +49,7 @@ public Response execute(String identityId, Request request) {
     }
 
     Set<String> scopes = Sets.newTreeSet(Splitter.on(" ").omitEmptyStrings().split(scope));
-    Optional<Authorization> possibleAuthorizationResponse = clientAuthorizationRepository.authorize(client, identityId, scopes, responseType);
+    Optional<Authorization> possibleAuthorizationResponse = clientAuthorizationRepository.authorize(client, identity.idenityId, identity.scopes, responseType);
```

Comment:
At this time there is no place where the user scopes(for bss-app) to be provided to the token, except the authorization request. The idea was these scopes to be stored for each user in the accouns app and when and provided to the api-gateway on when the accounts provides accessToken. 

With this change I move the decision which scopes to be provided out of the library. In that ResourceOwnerIdentityFInder.find is provided the request which should contains the requested scopes, clinetID and SID, so we have all the information that we need to decide and to check the requested scopes